generation/classifier_free_guidance/2d_ddpm_classifier_free_guidance_tutorial.ipynb

```diff
@@ -0,0 +1,959 @@
+{
```

Comment:
<p>I believe the definition for&nbsp;<code>guidance_scale</code>&nbsp;is in Reference [2]. It would be helpful to add a brief explanation here to direct users to obtain more information from [2].</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Done. </p>